must-gather/collection-scripts/gather_operator

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+BASE_COLLECTION_PATH="/must-gather"
+OSC_PATH=${BASE_COLLECTION_PATH}/sandboxed-containers/operator
+mkdir -p ${OSC_PATH}
+
+/usr/bin/oc describe csv -n openshift-sandboxed-containers-operator > ${OSC_PATH}/csv
+/usr/bin/oc describe subscription -n openshift-sandboxed-containers-operator > ${OSC_PATH}/subscription
```

Comment:
This also looks very similar to https://github.com/openshift/sandboxed-containers-operator/blob/master/must-gather/collection-scripts/gather_sandboxed_containers_logs#L11

Can you simply adopt that pattern and add it to the exactly same location as that? (with a different file name).

This makes me think a loop would benefit the code, making it cleaner.